### PR TITLE
feat: add pull request template

### PR DESCRIPTION
**Pull request by Brodypen**
Add basic pr template

### CODE REVIEW

**Review by Brodypen — COMMENTED** [review]
@Brodypen yippie

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Guide to linking issues here:
+https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
+-->
+
+**Added to documentation? 📝**
+
+- [ ] 📜 readme
+- [ ] 📜 other
+- [ ] 🙅 no documentation needed
+
```
Comment (bnleft) [inline .github/pull_request_template.md:39]: **suggestion:** remove

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+**What type of PR is this? (check applicable) 🚧**
+
+- [ ] ♻️ Refactor
+- [ ] ✨ Feature
+- [ ] 🐛 Bug Fix
+- [ ] 👷 Optimization
+- [ ] 📝 Documentation Update
+- [ ] 🔖 Release
+- [ ] 🚩 Other
+
```
Comment (bnleft) [inline .github/pull_request_template.md:10]: **suggestion:** remove

**Review by bnleft — CHANGES_REQUESTED** [review]
Description, Screenshots, Additional Context are the only sections we need

File: .github/pull_request_template.md (hunk 1 above)
Comment (Brodypen) [reply .github/pull_request_template.md:39]: Should we not link the pull request with the issue

Comment (bnleft) [reply .github/pull_request_template.md]: The `Development` section under `Milestone` section on the side bar to the right of each pull requests. This section can link the issue.

Comment (Brodypen) [reply .github/pull_request_template.md]: Very cool!

**Review by bnleft — APPROVED** [review]

**Pull request merged**